components/src/tooltips/useTooltip.ts

```diff
@@ -84,7 +84,7 @@ export function useTooltip(options: UseTooltipOptions = {}): UseTooltipResult {
     style: tooltipState.tooltipStyle,
     arrowStyle: tooltipState.arrowStyle,
     placement: tooltipState.placement,
-    arrowRef,
+    arrowRef: arrowRef,
```

Comment:
Accidental meaningless code change?

```suggestion
    arrowRef,
```